### ISSUE TITLE
Feature request-Polt rotate x axis labels

### ISSUE THREAD

**Issue by icekylin** [jasp-stats/jasp-issues#1453]
Often times it is useful to rotate the x axis labels to be vertical if there are too many labels and they overlap. This function accomplishes that and ensures the labels are horizontally centered relative to the tick line.

Consider enabling this function in the polts editing menu and adding corresponding buttons.

 Here is a good tool and example [https://github.com/daattali/ggExtra#rotatetextx---rotate-x-axis-labels](https://github.com/daattali/ggExtra#rotatetextx---rotate-x-axis-labels)

**Comment by kadewe** [comment]
Correct me, if I am wrong:

this needs "only" an input element for "angle" (number between 0 and 360) and an "angle" parameter sneaked into the axis.text.x?

Example:

library(ggplot2)
disprt <- data.frame(group = rep(c("AAAAAAAA", "BBBBBBBBBB"), 50), distances = rnorm(100))
ggplot(disprt, aes(group, distances)) + geom_boxplot() +
theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))

**Comment by tomtomme** [comment]
@kadewe 
I do not know. But if you have any chance to make a PR for jasps Plot Editor, that would be extremely cool!!!

**Comment by EJWagenmakers** [comment]
I would like to support this suggestion again, @vandenman 
See also https://forum.cogsci.nl/discussion/9542/distribution-plot#latest

**Comment by shun2wang** [comment]
I even tried to implement it by myself few weeks ago, see: https://github.com/shun2wang/jaspGraphs/tree/addRotateAxis

But the GUI design part has not been completed yet. It would be great if @vandenman could provide some help or suggestions!🍻

**Comment by vinschger** [comment]
Hi, is there any news on this? May I ask in general (I am not a developer and have no coding and no github experience), is there a roadmap/statusreport, so that I can check if reported bugs or feature requests are solved or what is their current status?

**Comment by tomtomme** [comment]
Yes, we need this. Lets Hope this will make it into the next Version. But I sadly cannot promised anything. There is so much to do! No real roadmap available, sorry. 
But you could Filter the issue list of Things that are listed as "in Progress" for 0.20

**Comment by vinschger** [comment]
thank you. how to filter for "in progress"? can I support the jasp project as a "private person"? you all do a great job for making jasp better with each update, thanks so much.

**Comment by tomtomme** [comment]
I am a private person too. There are many ways to support. Help with the issues here or translating (https://jasp-stats.org/translation-guidelines) are 2 ways to contribute without coding knowledge. Another way is to spread the word on youtube etc. or help answering stuff in the forum (https://jasp-stats.org/support/) or donate (https://jasp-stats.org/community/).

Filtering the list is done e.g. via https://github.com/jasp-stats/jasp-issues/issues?q=is%3Aopen+is%3Aissue+project%3Ajasp-stats%2F38+label%3A%22Feature+Request%22+sort%3Acomments-desc 

and then scrolling through the list and looking for linked pull requests
